charts/rancher-istio/1.4.900/questions.yaml

```diff
@@ -1,4 +1,3 @@
 labels:
-  rancher.istio.v1.4.700: 1.4.7
-rancher_min_version: 2.3.4-rc1
-rancher_max_version: 2.3.6
+  rancher.istio.v1.4.900: 1.4.9
+rancher_min_version: 2.3.7-rc1
```

Comment:
Changed min version to 2.3.4-rc1. I had missed this requirement in the issue, so now the PR meets all requirements.

Reply:
@deniseschannon want to confirm this ^, as this is a security release we want to go all the way back. Istio 1.4.8 only went back to 2.3.7 because it was non-security patch release.